pkgs/top-level/all-packages.nix

```
@@ -29757,6 +29757,8 @@ with pkgs;
 
   office-code-pro = callPackage ../data/fonts/office-code-pro { };
 
+  _0xproto = callPackage ../data/fonts/ohxproto { };
```

eval seems to complain about this

```suggestion
  _0xproto = callPackage ../data/fonts/0xproto { };
```

---

Sorry, seems I forgot to change the path there